### DISCUSSION TITLE
404

### DISCUSSION THREAD

**Discussion by SmartManoj** [pishangujeniya/clipsync-windows#3] [General]
http://clipsync.pishangujeniya.com/

**Comment by pishangujeniya** [comment]
@SmartManoj I have used the free tier of AWS, and now I am in search of one server to host my website, 
I am planning to switch my website from WordPress to Angular, and is currently a private repository and is under development, and also have the plan to use [Firebase Hosting](https://firebase.google.com/docs/hosting) for deployment.
Thanks @SmartManoj caring.
